subprojects/language-java/src/main/java/org/gradle/external/javadoc/CoreJavadocOptions.java

```diff
@@ -784,4 +785,16 @@ protected String getExtraOptions() {
                 .map(e -> e.getKey() + ":" + e.getValue())
                 .collect(Collectors.joining(", "));
     }
+
+    /**
+     * Internal access for the option file - do not use as this will be removed in a later version
```

Comment:
Why is it not `@Deprecated`, or emitting a deprecation warning?  Just to avoid patching tests for a small, temporary change?

Reply:
If this should not be used directly by users then I think it should at least have a `@Deprecated`